x/ibc/09-localhost/types/client_state.go

```diff
@@ -106,28 +129,24 @@ func (cs ClientState) VerifyConnectionState(
 	store sdk.KVStore,
 	cdc codec.BinaryMarshaler,
 	_ uint64,
-	prefix commitmentexported.Prefix,
+	_ commitmentexported.Prefix,
 	_ []byte,
 	connectionID string,
 	connectionEnd connectionexported.ConnectionI,
 ) error {
-	path, err := commitmenttypes.ApplyPrefix(prefix, host.ConnectionPath(connectionID))
```

Comment:
I think this requires a spec change too. @cwgoes

Reply:
if not, document why the implementation differs from spec

Reply:
We do not need a prefix because we are already being passed in the clientID prefixed store into our function.

So we shouldn't be prefixing at all with `ibc`. We really only need that prefix in the specific clients when we are construcing the proof path. It's not needed to get from the `clientStore`.

Reply:
this is good info for when we add a `spec` dir for locahost since this differs from the ICS spec in implementation but not semantics

Reply:
Aye, excellent use-case for a `spec` directory.